src/app/pages/RewardPage/components/LiquidTab/index.tsx

```diff
@@ -20,19 +20,14 @@ export const LiquidTab: React.FC<ILiquidTabProps> = ({
   lastWithdrawalInterval,
 }) => {
   const { t } = useTranslation();
+  const { data: stakingRewardEvents } = useGetRewardEarnedEvents();
 
-  const { events: stakingRewardEvents } = useGetContractPastEvents(
-    'stakingRewards',
-    'RewardWithdrawn',
-  );
-
-  const totalRewardsEarned = useMemo(
-    () =>
-      stakingRewardEvents
-        .map(item => item.returnValues.amount)
-        .reduce((prevValue, curValue) => prevValue.add(curValue), bignumber(0)),
-    [stakingRewardEvents],
-  );
+  const totalRewardsEarned = useMemo(() => {
+    return stakingRewardEvents?.rewardsEarnedHistoryItems.reduce(
+      (previousItem, currentItem) => previousItem.add(currentItem.amount),
+      bignumber(0),
+    );
```

Comment:
Same `reduce` signature used across feature for adding up amounts, you could extract it it to the helper function:
```
const addAmounts = (items: Items[]) => items.reduce((p, c) => p.add(c.amount), bignumber(0));
...
addAmounts(stakingRewardEvents?.rewardsEarnedHistoryItems || []);
addAmounts(lendingRewardsData?.rewardsEarnedHistoryItems || []);
...
```